exercises/strings/countLetter/countLetter.js

```diff
@@ -23,11 +23,23 @@
 
 function countLetter(string, letter) {
   // This is your job. :)
+  var i;
```

Comment:
You don't need to declare variables before they're used a la C/C++. You can do:

```js
for (var i = 0; i < string.length; i++) {
  // ...
}
```

Also, consider using a `for ... of` loop here since you don't use the index for anything other than bookkeeping, e.g.,

```js
function countLetter(string, letter) {
  let count = 0;

  for (let char of string) {
    if (char === letter) {
      count += 1;
    }
  }

  return count;
}
```